src/assets/scripts/cipher-web-worker.js

```diff
@@ -0,0 +1,17 @@
+importScripts('main.js');
+
+onmessage = function(e) {
+  try {
+    if (e.data.operation == 0) {
```

Comment:
This would be much better represented as a `switch-case-default` as it then makes it immediately obvious that the choice of action is occurring based on the value of "value" and not some arbitrary test.